eslint-plugin-prettier.js

```diff
@@ -73,21 +74,12 @@ function reportDifference(context, difference) {
 //  Module Definition
 // ------------------------------------------------------------------------------
 
-/**
- * @type {Plugin}
- */
+// Type hint configs key as non-nullable to avoid type-checking errors in when
+// assigning to eslintPluginPrettier.config below
+/** @type {Plugin & {configs: NonNullable<Plugin['configs']>}} */
```

Comment:
```suggestion
/**
 * Type hint `configs` key as non-nullable to avoid type-checking errors in when
 * assigning to `eslintPluginPrettier.configs` below
 *
 * @type {Plugin & {configs: NonNullable<Plugin['configs']>}}
 */
```

Avoid mixed comments

Reply:
Fixed